src/charm.py

```diff
@@ -740,6 +721,144 @@ def _gen_wp_config(self):
         )
         return "\n".join(wp_config)
 
+    def _container(self):
+        return self.unit.get_container("wordpress")
```

Comment:
Do you think a docstring here and in any other functions being added in the PR would be helpful?

Reply:
Added, thanks!